tutorindigo/templates/indigo/lms/templates/header/navbar-authenticated.html

```diff
@@ -53,7 +53,23 @@
       </div>
     % endif
   </div>
-  <div class="secondary">
+  <div class="secondary d-flex align-items-center">
```

Comment:
Move this code to a separate template file and replace this by an `<% include ... />` statement, such that we don't duplicate code (same in navbar-not-authenticated.html).